web3rpc/rpc-specs/paths/klay/miscellaneous/getAllRecordsFromRegistry.yaml

```diff
@@ -0,0 +1,99 @@
+
+      parameters:
+        - name: name
+          description: Name of which active system contract registered with.
+          required: true
+          schema:
+            type: string
+          in: query
+        - name: blockNumberOrHashOrTag
+          in: query
+          description: Integer or hexadecimal block number, or the string "earliest", "latest" or "pending" as in the default block parameter, or block hash.
+          required: true
+          schema:
+            $ref: "../../../components/schemas/common/Common.yaml#/components/schemas/BlockNumberOrHashOrTag"
+
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: "../../../components/requests/JsonRpcRequest.yaml#/components/schemas/JsonRpcRequest"
+                - $ref: "#/components/schemas/KlaygetAllRecordsFromRegistryReq"
+
+      responses:
+        200:
+          description: Returns the account key or an empty key value.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "../../../components/responses/JsonRpcResponse.yaml#/components/schemas/JsonRpcResponse"
+                  - $ref: "#/components/schemas/KlaygetAllRecordsFromRegistryResp"
+
+      x-codeSamples:
+        - lang: "curl"
+          label: "Curl"
+          source:
+            $ref: "../../../code-samples/curl/klay/miscellaneous/getAllRecordsFromRegistry.sh"
+        - lang: "java"
+          label: "Java"
+          source:
+            $ref: "../../../code-samples/java/src/main/java/opensdk/sdk/apis/klay/miscellaneous/KlayGetAllRecordsFromRegistryExample.java"
+        - lang: "javascript"
+          label: "Javascript"
+          source:
+            $ref: "../../../code-samples/javascript/klay/miscellaneous/getAllRecordsFromRegistry.js"
+        - lang: "python"
+          label: "Python"
+          source:
+            $ref: "../../../code-samples/python/klay/miscellaneous/getAllRecordsFromRegistry.py"
+components:
+  schemas:
+    KlaygetAllRecordsFromRegistryReq:
+      type: object
+      required:
+        - method
+        - params
+      properties:
+        method:
+          type: string
+          default: 'klay_getAllRecordsFromRegistry'
+        params:
+          type: array
+          items:
+            properties:
+              Name:
+                type: string
+                description: Name of which active system contract registered with.
+              BlockNumberOrHashOrTag:
+                $ref: "../../../components/schemas/common/Common.yaml#/components/schemas/BlockNumberOrHashOrTag"
+          example:  ["kip113", "latest"]
+
+    KlaygetAllRecordsFromRegistryResp:
+      type: object
+      properties:
+        result:
+            type: string
+            description: Returns address of active system contract registered with name if exists.
+            example: "0x0000000000000000000000000000000000000402"
```

Comment:
Please check the actual response from the RPC!
`kip113` doesn't work. `KIP113` works. and the return type is an array of object.

```
> klay.getAllRecordsFromRegistry("KIP113", "latest")
[{
    activation: 0,
    addr: "0x3e80e75975bdb8e04b800485dd28bebec6d97679"
}]
```